wagtail/admin/userbar.py

```diff
@@ -144,6 +187,7 @@ def get_axe_configuration(self, request):
             "context": self.get_axe_context(request),
             "options": self.get_axe_options(request),
             "messages": self.get_axe_messages(request),
+            "customAltTextRuleConfig": self.get_axe_custom_alt_text_rule(request),
```

Comment:
Let’s change this so we provide a `spec` object (or similar) that would allow configuring more aspects of `axe.configure`.